Tools/CISupport/ews-build/steps.py

```diff
@@ -5923,6 +5928,32 @@ def getResultSummary(self):
         return {'step': configuration}
 
 
+class PrintClangVersion(shell.ShellCommandNewStyle):
+    name = 'print-clang-version'
+    haltOnFailure = False
+    flunkOnFailure = False
+    warnOnFailure = False
+
+    def __init__(self, **kwargs):
+        super().__init__(logEnviron=False, **kwargs)
+
+    @defer.inlineCallbacks
+    def run(self):
+        self.log_observer = logobserver.BufferLogObserver()
+        self.addLogObserver('stdio', self.log_observer)
+        self.command = ['../llvm-project/build/bin/clang', '--version']
```

Comment:
Nit: Can move this above as a class variable to be more consistent with rest of the code.